<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pyth IDE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="output_box">
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPlainTextEdit" name="output_text_edit">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="code_box">
        <property name="title">
         <string>Code</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="1">
          <widget class="QLabel" name="code_length_label">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code length:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTabWidget" name="code_tabs">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="input_box">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTabWidget" name="input_tabs">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="regular_input">
            <attribute name="title">
             <string>Regular Input</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPlainTextEdit" name="input_text_edit">
               <property name="font">
                <font>
                 <family>Monospace</family>
                </font>
               </property>
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="test_suite">
            <attribute name="title">
             <string>Test Suite</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="test_suite_label">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Lines / Test: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="test_suite_spinbox">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="test_suite_spacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0" colspan="3">
              <widget class="QPlainTextEdit" name="test_suite_text_edit">
               <property name="tabChangesFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="doc_box">
      <property name="title">
       <string>Pyth Character Reference</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="find_line_edit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="placeholderText">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="doc_table_widget">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCode_Tabs">
    <property name="title">
     <string>Code Tabs</string>
    </property>
    <addaction name="action_open_tab"/>
    <addaction name="action_duplicate"/>
    <addaction name="action_close_tab"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_debug"/>
    <addaction name="action_multi_line"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_import_heroku"/>
    <addaction name="action_heroku"/>
    <addaction name="action_to_clipboard"/>
    <addaction name="action_define_template"/>
   </widget>
   <addaction name="menuRun"/>
   <addaction name="menuExport"/>
   <addaction name="menuCode_Tabs"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_run">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Debug</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Return</string>
   </property>
  </action>
  <action name="action_open_tab">
   <property name="text">
    <string>Open a new Tab</string>
   </property>
   <property name="toolTip">
    <string>Open a new Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_close_tab">
   <property name="text">
    <string>Close current Tab</string>
   </property>
   <property name="toolTip">
    <string>Close current Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_to_clipboard">
   <property name="text">
    <string>Export to Clipboard</string>
   </property>
  </action>
  <action name="action_heroku">
   <property name="text">
    <string>Switch to Heroku-App</string>
   </property>
  </action>
  <action name="action_define_template">
   <property name="text">
    <string>Define Template for Export</string>
   </property>
  </action>
  <action name="action_import_heroku">
   <property name="text">
    <string>Import from Heroku-App</string>
   </property>
  </action>
  <action name="action_duplicate">
   <property name="text">
    <string>Duplicate current Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_multi_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Multi-Line Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
